javascript/src/UploadField.js

```diff
@@ -430,13 +430,14 @@ $.entwine('ss', function($) {
 			}
 
 			if (iframe.attr('src') == 'about:blank') {
+				var disabled = this.siblings();
```

Comment:
@scott1702 I was wondering the same thing actually. It wasn't linting, but rather something around the asset work which snuck into this PR